packages/shared/src/modules/project-groups/libs/enums/project-groups-api-path.enum.ts

```diff
@@ -1,5 +1,6 @@
 const ProjectGroupsApiPath = {
-	$ID: "/:id",
+	$ID: ":id",
+	$PROJECT_ID_$ID: "/:projectId/:id",
```

Comment:
what if project id passed doesn't match project group? this project id should be extracted on the backend side from project group
```suggestion
```

Reply:
I simplified it because of this functional
![image](https://github.com/user-attachments/assets/ceab878a-ae11-4871-ba5c-3fbaee072d3a)

The third parameter takes project id. On the other hand, I have to copy the functionality of the whole `checkUserPermissions` function somewhere else after taking the project id from the group,  and somehow pass the permissions from the route. I'd rather add another check to the service, and if project group is not included to the project return `NO_PERMISSION`

Reply:
in this case let's pass projectId to payload

Reply:
I just realised that DELETE method doesn't have body and that's why I pass it in params.

Reply:
and I'm trying to implement it, so it doesn't matter how you pass `project id` (but we still need to take it in router)
```
	public async deleteWithCheckingProject(
		id: number,
		projectId: number,
	): Promise<boolean> {
		const projectGroup = await this.projectGroupRepository.find(id);

		if (!projectGroup) {
			throw new ProjectGroupError({
				message: ExceptionMessage.PROJECT_GROUP_NOT_FOUND,
				status: HTTPCode.NOT_FOUND,
			});
		}

		const { projectId: currentProject } = projectGroup.toObject();

		if (currentProject.id !== projectId) {
			throw new ProjectGroupError({
				message: ExceptionMessage.NO_PERMISSION,
				status: HTTPCode.FORBIDDEN,
			});
		}

		return await this.projectGroupRepository.delete(id);
	}
```

Reply:
maybe we can remove this hooks for project group and calculate them inside service? I think this will be a more secure and right option